deploy/docker-compose-weave/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - external
       - secure
       - internal
+      - backoffice
```

Comment:
Yes. Exactly. We expect to have a monitoring tool looking at the individual services.